docker/skr/Dockerfile.skr

```diff
@@ -1,11 +1,10 @@
 FROM alpine:3.17.1
 
-RUN apk update && apk add curl
+RUN apk update && apk add --no-cache curl
 
 COPY ./bin/skr ./bin/get-snp-report /bin/
 COPY skr.sh tests/*_client.sh tests/skr_test.sh /
-RUN mkdir -p /tests/skr; mv *_client.sh /tests/skr; mv skr_test.sh /tests/skr
-RUN chmod +x /*.sh /tests/skr/*.sh; date > /made-date
+RUN mkdir -p /tests/skr; mv *_client.sh /tests/skr; mv skr_test.sh /tests/skr && chmod +x /*.sh /tests/skr/*.sh

```

Comment:
You should probably do this in the Dockerfile.debug too (if applicable)

Reply:
I'm not worried about the total number of layers in the debug version.  This was to reduce the number of layers to work around a layer limit present in the kata-cc uvm.